scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError
+            HTTP, GraphQL, UserError, CheckoutError
```

Comment:
I like `NativePaymentProcessingError` since it describes where the error is coming from an in what case.